db/migrate/20210316154530_create_appointment_reminders.rb

```diff
@@ -0,0 +1,12 @@
+class CreateAppointmentReminders < ActiveRecord::Migration[5.2]
```

Comment:
I spent far too much time thinking about whether this should also have a reference to patients. Ultimately, I don't think we need `Patient.appointment_reminders` so I left it out. It's essentially a has_many_through appointments relationship already, so it's easily discoverable.